ui/ozone/public/ozone_switches.cc

```diff
@@ -15,4 +15,8 @@ const char kOzoneDumpFile[] = "ozone-dump-file";
 // Try to enable drm atomic. This works only with drm platform.
 const char kEnableDrmAtomic[] = "enable-drm-atomic";
 
+// Try to enable wayland input method editor.
+// This works only with wayland platform.
```

Comment:
I think this second line could be omit. this first lines implicitly says it, and so does the command line name itself.